pkgs/tools/security/der-ascii/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+let
+
+pname = "der-ascii";
+version = "0.1.0";
+
+in
+
+buildGoModule {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1my93m1rx08kn2yms6k8w43byr8k61r1nra4b082j8b393wwxkqc";
+  };
+  vendorSha256 = null;
+
+  ldflags = [ "-s" "-w" ];
+
+  meta = with lib; {
+    description = ''
+      A small human-editable language to emit DER or BER encodings of ASN.1
+      structures and malformed variants of them
+    '';
+    homepage = "https://github.com/google/${pname}";
```

In review for other derivations I've been encouraged to not use variable indirection for the `homepage` since it seldom changes and makes it harder to click this as a valid URL.

```suggestion
    homepage = "https://github.com/google/der-ascii";
```

---

Yeah, I was a bit dubious here. Good to know, thanks!